File: official/docs/golang/current/addresses/verify.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v5"
)

func verify() {
	client := easypost.New("EASYPOST_API_KEY")

	address, _ := client.VerifyAddress("adr_...")

	fmt.Println(address)
}

File: official/docs/java/current/addresses/verify.java
package addresses;

import com.easypost.exception.EasyPostException;
import com.easypost.model.Address;
import com.easypost.service.EasyPostClient;

import java.util.HashMap;

public class Verify {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient("EASYPOST_API_KEY");

        Address address = client.address.verify("adr_...");

        System.out.println(address);
    }
}
